content/guides/cloud/smart-orchestration.md

```diff
@@ -87,7 +87,9 @@ Continuous Integration (CI) pipelines are typically costly processes that can
 demand significant compute time. **When a test failure occurs in CI, it often
 does not make sense to continue running the remainder of a test suite** since
 the process has to start again upon merging of subsequent fixes and other code
-changes.
+changes. When Auto Cancellation is enabled, once the number of failed tests goes
+over a preset threshold, the entire test run is cancelled. Note that any
```

Comment:
```suggestion
over a preset threshold, the entire test run is canceled. Note that any
```
We've used `canceled` throughout the docs, and that aligns with American English convention.